src/components/manager/dropBox/DropBoxJsonSelect.tsx

```diff
@@ -28,44 +28,59 @@ const InnerDropBoxJsonSelect = ({
   );
 };
 
+const EXISTING = "existing";
+const NEW = "new";
+const NONE = "none";
+type DROP_BOX_SELECT_TYPE = typeof EXISTING | typeof NEW | typeof NONE;
```

Comment:
types are generally done not in all uppercase, so i'd expect this to be more like `DropBoxSelectType`

Reply:
maybe a string enum would work? https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Reply:
good idea